index.html

```diff
@@ -249,70 +284,78 @@ <h2 class="text-center">Languages</h2>
         </ul>
       </div>
     </div>
-
-    <div class="info-box framework">
-      <div class="info-content text-dark">
-        <h2 class="text-center">Frameworks</h2>
-        <ul class="framework-list framework-list-1 w-full">
-          <li>React.js</li>
-          <li>Ruby</li>
-          <li>RSpec</li>
-        </ul>
-        <ul class="framework-list framework-list-2 w-full">
-          <li>Capybara</li>
-          <li>Selenium</li>
-        </ul>
+    <div class="box flex flex-col">
+      <div class="box-img">
+        <img class="w-full h-full" src="./assets/images/img-placeholder.png" alt="imgPlaceholder" />
+      </div>
+      <div class="box-content fw-600">
+        <h2 class="">Multi-Post Stories Gain + Glory</h2>
+        <div class="tags">
+          <ul class="flex w-full justify-between">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+        </div>
+        <div class="flex justify-center">
+          <button class="text-center btn text-white" type="button">
+            See Project
+          </button>
+        </div>
       </div>
     </div>
-
-    <div class="info-box skill">
-      <div class="info-content text-dark">
-        <h2 class="text-center">Skills</h2>
-        <ul class="skill-list skill-list-1 w-full">
-          <li>Database Management</li>
-          <li>Version Control</li>
-        </ul>
-        <ul class="skill-list skill-list-2 w-full">
-          <li>CLI</li>
-          <li>Web Development</li>
-          <li>API Design</li>
-        </ul>
+    <div class="box flex flex-col">
+      <div class="box-img">
+        <img class="w-full h-full" src="./assets/images/img-placeholder.png" alt="imgPlaceholder" />
+      </div>
+      <div class="box-content fw-600">
+        <h2 class="">Multi-Post Stories Gain + Glory</h2>
+        <div class="tags flex w-full justify-between">
+          <ul class="flex w-full justify-between">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+        </div>
+        <div class="flex justify-center">
+          <button class="text-center btn text-white" type="button">
+            See Project
+          </button>
+        </div>
       </div>
     </div>
   </section>
 
-  <section id="contact-section" class="contact-section relative">
+  <section id="contact-section" class="contact-section relative bg-white">
     <div class="contact-container">
       <div class="contact-header">
         <h2 class="">
           I'm always interested in hearing about new projects, so if you'd
           like to chat please get in touch.
         </h2>
```

Comment:
- [ ] [OPTIONAL] I'll make this optional because this project requirements is about modals, and this is contact section.
- [ ] I kindly suggest that you make your contact section the same as the Figma design:
- [ ] Kindly put your contact form and not a project as the right side of your contact section.


|   |  |
| ------------- | ------------- |
| Your Design  | <img alt="image" width="905" src="https://user-images.githubusercontent.com/52934128/239587475-b745c8bf-114c-4e90-8edd-1a5312468aec.png">  |
| Figma Design  | <img alt="image" width="599" src="https://user-images.githubusercontent.com/52934128/239587739-685ac896-3dee-4e13-ad96-1baad391c160.png">  |